### ISSUE TITLE
[bug]: list_checkpoint_configs fails when INVOKEAI_ROOT is a symlink

### ISSUE THREAD

**Issue by keturn** [invoke-ai/InvokeAI#5224]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Linux

### GPU

cuda

### VRAM

12GB

### What version did you experience this issue on?

d0464a57930f8a2d2e37f56ea58b75a258573a50

### What happened?

There's a recurring error when I go to the web UI for the model manager:

```
  File "InvokeAI/invokeai/app/api/routers/models.py", line 344, in list_ckpt_configs                                                                       
    return ApiDependencies.invoker.services.model_manager.list_checkpoint_configs()                                                                                         
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                         
  File "InvokeAI/invokeai/app/services/model_manager/model_manager_default.py", line 389, in list_checkpoint_configs                                       
    return [(conf_path / x).relative_to(root_path) for x in conf_path.glob("**/*.yaml")]                                                                                    
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                                                    
  File "InvokeAI/invokeai/app/services/model_manager/model_manager_default.py", line 389, in <listcomp>                                                    
    return [(conf_path / x).relative_to(root_path) for x in conf_path.glob("**/*.yaml")]                                                                                    
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                    
  File "python3.11/pathlib.py", line 730, in relative_to                                                                            
    raise ValueError("{!r} is not in the subpath of {!r}"                             
ValueError: '/the-real-absolute-path/InvokeAI/configs/stable-diffusion/v2-inference-v.yaml' is not in the subpath of '/path-containing-symlink/InvokeAI' OR one path is relative and the other is absolute.                                                                    
```

Problem seems to stem from the fact that I added a new filesystem and moved my INVOKEAI_ROOT there, then put a symlink in the old place so that any references to those old paths would keep working.

**Comment by lstein** [comment]
@keturn I wasn't able to exactly reproduce your bug, but after making an INVOKEAI_ROOT symlink I did encounter similar behavior in other model manager functions that list models. I made a small fix which solved these problems; could you try PR #5332 and see if it fixes your issue?

**Comment by keturn** [comment]
confirmed I no longer see any errors of this sort in current `main` (with #5332 merged)
